src/widget/form/profileform.cpp

```diff
@@ -247,10 +247,14 @@ void ProfileForm::onRenameClicked()
     Nexus& nexus = Nexus::getInstance();
     QString cur = nexus.getProfile()->getName();
     QString title = tr("Rename \"%1\"", "renaming a profile").arg(cur);
+
     do
     {
-        QString name = QInputDialog::getText(this, title, title+":");
-        if (name.isEmpty()) break;
+        QString name = QInputDialog::getText(0, title, title + ":");
```

Comment:
How do you think, a widget is centred on it's parent, when there is none?. See for yourself:
![bildschirmfoto von 2015-08-06 09 27 45](https://cloud.githubusercontent.com/assets/440517/9106079/a855373a-3c1d-11e5-9921-bfeee3d4a503.png)

Again, this is no solution for the actual problem here.